pkg/controller/deployment_inplace.go

```diff
@@ -273,6 +273,11 @@ func (dc *controller) reconcileOldMachineSetsInPlace(ctx context.Context, allMac
 		return true, nil
 	}
 
+	// Manual inplace update
+	if deployment.Spec.Strategy.InPlaceUpdate.OrchestrationType == v1alpha1.OrchestrationTypeManual {
+		return true, nil
```

Comment:
Since we return `true, nil` here, this would mean that in the caller the `if MachineDeploymentComplete(d, &d.Status) {...` block will not be called and ig eventually we would like that to be called, no?
I do not see a way that this block will eventually be called as well except [here](https://github.com/gardener/machine-controller-manager/blob/master/pkg/controller/deployment_inplace.go#L259)
Am I missing anything?

Reply:
That's correct, I changed it to return false. Thanks for pointing it out.